### PR TITLE
Update SAML query profile documentation with the new flow

### PR DESCRIPTION
**Pull request by deshankoswatte**
Fixes: https://github.com/wso2/docs-is/issues/924

### CODE REVIEW

File: en/docs/learn/querying-saml-assertions.md [hunk 1]
```diff
@@ -32,29 +32,23 @@ authority.
 !!! note "Before you begin"
```
Comment (YvonneW) [inline en/docs/learn/querying-saml-assertions.md]: !!! tip

File: en/docs/learn/querying-saml-assertions.md [hunk 2]
```diff
@@ -32,29 +32,23 @@ authority.
...
 
     1.  Clone or download the client application from [this GitHub
-        location](https://github.com/gayangithub/wso2-is-assertion-query-client)
-        and extract it. This folder location will be referred to as
+        location](https://github.com/wso2/samples-is),
+        extract it and navigate to
```
Comment (YvonneW) [inline en/docs/learn/querying-saml-assertions.md]: extract it,

File: en/docs/learn/querying-saml-assertions.md [hunk 3]
```diff
@@ -32,29 +32,23 @@ authority.
...
+        `           sample-saml-query-profile/saml-query-profile-client          `.
+        This folder location will be referred to as
         `            <CLIENT_HOME>           ` in this document.
```
Comment (YvonneW) [inline en/docs/learn/querying-saml-assertions.md]: Hereafter, this folder location will be referred to as `<CLIENT_HOME>`.

File: en/docs/learn/querying-saml-assertions.md [hunk 4]
```diff
@@ -32,29 +32,23 @@ authority.
...
 
     2.  Build the client application by executing the following command on
```
Comment (YvonneW) [inline en/docs/learn/querying-saml-assertions.md]: To build the client application, execute the following command in a command prompt.

File: en/docs/learn/querying-saml-assertions.md [hunk 5]
```diff
@@ -32,29 +32,23 @@ authority.
...
-        the `             <CLIENT_HOME>            ` root folder.
+        the `             <CLIENT_HOME>            ` folder.
 
         ``` java
         mvn clean install
         ```
 
 1.  Start the WSO2 Identity Server and log in to the management console.
```
Comment (YvonneW) [inline en/docs/learn/querying-saml-assertions.md]: Sign in to WSO2 Identity Server Management Console.

File: en/docs/learn/querying-saml-assertions.md [hunk 6]
```diff
@@ -32,29 +32,23 @@ authority.
...
-3.  Click **Import Cert** to import a certificate to the
-    **wso2carbon.jks** keystore.  
-    ![import-cert-keystore](../assets/img/tutorials/import-cert-keystore.png)
-4.  Upload the certificate found in
-    `          <CLIENT_HOME>/src/test/resources/soa.cert         ` and
-    click **Import**.
-5.  Click **Add** under **Service Providers** on the **Main** tab.
-6.  Configure a service provider for SAML and click **Update**.  
-    To try querying an assertion using the a sample application,
+2.  Click **Add** under **Service Providers** on the **Main** tab.
```
Comment (YvonneW) [inline en/docs/learn/querying-saml-assertions.md]: On **Main** tab, click **Identity > Service Providers > Add**.

File: en/docs/learn/querying-saml-assertions.md [hunk 7]
```diff
@@ -32,29 +32,23 @@ authority.
...
+3.  Configure a service provider for SAML and click **Update**.
+    To try querying an assertion using the sample application,
     configure the service provider for the travelocity sample
     application and follow the steps in the
     [try out scenario](#try-out-scenario) section.
```
Comment (YvonneW) [inline en/docs/learn/querying-saml-assertions.md]: To create a service provider that can be used to query an assertion, see [Try out scenario](#try-out-scenario).

File: en/docs/learn/querying-saml-assertions.md (hunk 1 above)
Comment (deshankoswatte) [reply en/docs/learn/querying-saml-assertions.md]: Fixed

File: en/docs/learn/querying-saml-assertions.md (hunk 2 above)
Comment (deshankoswatte) [reply en/docs/learn/querying-saml-assertions.md]: Fixed

File: en/docs/learn/querying-saml-assertions.md (hunk 3 above)
Comment (deshankoswatte) [reply en/docs/learn/querying-saml-assertions.md]: Fixed

File: en/docs/learn/querying-saml-assertions.md (hunk 4 above)
Comment (deshankoswatte) [reply en/docs/learn/querying-saml-assertions.md]: Fixed

File: en/docs/learn/querying-saml-assertions.md (hunk 5 above)
Comment (deshankoswatte) [reply en/docs/learn/querying-saml-assertions.md]: Fixed

File: en/docs/learn/querying-saml-assertions.md (hunk 6 above)
Comment (deshankoswatte) [reply en/docs/learn/querying-saml-assertions.md]: Fixed

**Pull request merged**